<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Car" name="carWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="WelcomeScreen" name="welcomeWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="Level1" name="level1Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="LevelSelectScreen" name="levelSelectWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="level2Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="level3Widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="FailScreen" name="failScreen" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <zorder>welcomeWidget</zorder>
   <zorder>level1Widget</zorder>
   <zorder>levelSelectWidget</zorder>
   <zorder>level2Widget</zorder>
   <zorder>level3Widget</zorder>
   <zorder>carWidget</zorder>
   <zorder>failScreen</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Car</class>
   <extends>QWidget</extends>
   <header>car.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WelcomeScreen</class>
   <extends>QWidget</extends>
   <header>welcomescreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Level1</class>
   <extends>QWidget</extends>
   <header>level1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LevelSelectScreen</class>
   <extends>QWidget</extends>
   <header>levelselectscreen.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FailScreen</class>
   <extends>QWidget</extends>
   <header>failscreen.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
